plugin/storage/es/esCleaner.py

```diff
@@ -46,7 +47,10 @@ def main():
     if str2bool(os.getenv("ARCHIVE", 'false')):
         filter_archive_indices(ilo, prefix)
```

Comment:
nit: based on whether archive only works with rollover, then for consistency might be good to add `_rollover` suffix to function - and may be check for `ROLLOVER` and if not set, then print a warning message?